tests/acceptance/features/apiGraph/getGroup.feature

```diff
@@ -0,0 +1,51 @@
+@api @skipOnOcV10
+Feature: get groups and their members
+  As an admin
+  I want to be able to get groups
+  So that I can see all the groups and their members
+
+  Background:
+    Given user "Alice" has been created with default attributes and without skeleton files
+    And the administrator has given "Alice" the role "Admin" using the settings api
+
+
+  Scenario: admin user lists all the groups
+    Given group "tea-lover" has been created
+    And group "coffee-lover" has been created
+    And group "h2o-lover" has been created
+    When user "Alice" gets all the groups using the Graph API
+    Then the extra groups returned by the API should be
+      | tea-lover    |
+      | coffee-lover |
+      | h2o-lover    |
+
+
+  Scenario: normal user cannot get the groups list
+    Given user "Brian" has been created with default attributes and without skeleton files
+    And group "tea-lover" has been created
+    And group "coffee-lover" has been created
+    And group "h2o-lover" has been created
+    When user "Brian" gets all the groups using the Graph API
+    Then the HTTP status code should be "401"
```

Comment:
do we need to check if the request body was empty for this? needed? what do you think?

Reply:
we have that step in core tests. Yeah, good to have them here as well.